index.rst

```diff
@@ -1,29 +1,57 @@
-:tocdepth: 1
+:tocdepth: 2
 
 .. sectnum::
 
 .. Metadata such as the title, authors, and description are set in metadata.yaml
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
+.. warning::
 
    **This technote is a work-in-progress.**
 
 Abstract
 ========
 
-This technote summarizes the groups and individuals making in-kind contributions to SIT-Com
+This technote summarizes the groups and individuals making in-kind contributions to the Vera C. Rubin Observatory System Integration, Test, and Commissioning (SIT-Com) effort.
+
+Introduction
+============
+
+The Vera C. Rubin Observatory recognizes the individuals listed below making value-added non-financial contributions to the System Integration, Test, and Commissioning (SIT-Com) effort in order to facilitate an efficient transition to LSST Operations.
+By sharing their technical and scientific expertise, these individuals enhance and diversify the Project's planned commissioning effort.
+The named participants will work directly alongside Rubin Observatory staff and will have access to commissioning data products as they are acquired in order to complete their assignments.
+
+The participating individuals were invited to join the Rubin Observatory SIT-Com effort through one of two programs:
+
+1. US/Chile Commissioning Announcement of Opportunity Program and
+2. International In-Kind Contribution Program where the recipient group is the “Rubin Commissioning Team”.
+
+The two types of contribution are being managed by Rubin Observatory as a combined resource.
+As indicated below, the in-kind contributions are organized into discrete groups of like interests and skills with a designated point of contact.
+We anticipate that participants will engage in multiple aspects of commissioning according to their abilities and the schedule and priorities of Rubin Observatory.
+
+The deep level of engagement described in these programs comes with specific rights and responsibilities, as outlined https://sitcomtn-010.lsst.io/ (Section 4.1).
+Working as part of the SIT-Com team offers direct experience with and deep understanding of the full chain from observations to final data products and data access tools that will be released to the science community, including the hardware, image properties, and Science Pipeline algorithms.
+Through their efforts, these individuals will contribute to the readiness of Rubin Observatory to deliver its science products to the global Rubin community.
+
+The Project will not rely on the contributions from non-Rubin-staff team members to fulfill core construction requirements and operational readiness criteria.
```

Comment:
This paragraph feels a bit out of place and is a bit of a repeat of the first.
I think there are a few ways to better integrate this.

1. Just add a softer transition. Consider: "It should be emphasized that the Project will not...." 
2. Move line 38 into the intro paragraph, nix line 39 as it is also in the intro, then add a "It should be noted that no papers...."